pkg/network/state.go

```diff
@@ -551,6 +565,31 @@ func (ns *networkState) mergeByCookie(conns []ConnectionStats) ([]ConnectionStat
 	return conns, connsByKey
 }
 
+// getClientCapacityPercentage calculates the current capacity percentage (0-100) for a client's closed connections buffer
+func (ns *networkState) getClientCapacityPercentage(client *client) float64 {
+	if ns.maxClosedConns == 0 {
```

Comment:
I think this would be a fatal condition, we shouldn't ever see this.

Reply:
removed